iceoryx_posh/include/iceoryx_posh/internal/popo/ports/server_port_roudi.hpp

```diff
@@ -62,6 +65,8 @@ class ServerPortRouDi : public BasePort
     const MemberType_t* getMembers() const noexcept;
     MemberType_t* getMembers() noexcept;
 
+    void handleCaProProtocollViolation(iox::capro::CaproMessageType messageType) const noexcept;
```

Comment:
I know it states do not review therefore will not tell you that Protocol is only written with a single `l`
And that your argument is missing some const.